src/applications/login/tests/containers/MockAuth.unit.spec.js

```diff
@@ -24,20 +67,29 @@ describe('MockAuthButton', () => {
   });
 
   Object.values(environments).forEach(currentEnvironment => {
-    it('should take you to the right link when clicked', () => {
+    it('should take you to the right link when clicked', async () => {
       process.env.BUILDTYPE = currentEnvironment;
-      const { container } = render(<MockAuthButton />);
-      const correctLink = `${
-        ENVIRONMENT_CONFIGURATIONS[process.env.BUILDTYPE].API_URL
-      }/v0/sign_in/authorize?client_id=vamock`;
-      const button = $('.mauth-button', container);
-      fireEvent.click(button);
-      expect(window.location).to.equal(correctLink);
+      const tree = SkinDeep.shallowRender(<MockAuthButton />);
+      const button = tree.subTree('.mauth-button');
+      if (
+        currentEnvironment === environments.LOCALHOST ||
+        currentEnvironment === environments.VAGOVDEV
```

Comment:
Same thing here, if we're checking both the same types of things might as well consolidate

`[env.LOCAL, env.DEV].includes(currentEnv)`